deployment/ansible/group_vars/all

```diff
@@ -3,7 +3,7 @@ postgresql_support_libpq_version: "9.4.0-1.pgdg14.04+1"
 app_username: "vagrant"
 
 # open trip planner vars
-otp_process_mem: 4G
+otp_process_mem: 2G
```

Comment:
Neat, good to know.  However much we need to do the build should be constant, but I'll keep that in mind for future reference.